### DISCUSSION TITLE
Workflow Trigger Order

### DISCUSSION THREAD

**Discussion by tvc12** [Tracardi/tracardi#870] [Q&A]
Hi Tracardi team,
I have event **purchase**. I have some flows named:

1. Copy Data To Profile
2. Tagging Elite Profile
3. Merge Profile
4. Send email

I want the 4 workflows to run in the correct order.
I don't want to combine all 4 workflows into one large workflow. Is there a solution for this?

**Comment by atompie** [comment]
@tvc12 Unfortunately there is no way to set the order of workflow triggering.

**Comment by tvc12** [comment]
any solution for workaround? @atompie

**Reply by atompie** [reply]
Step 1 and 3 do not have to be done in workflow. If merging is on email then it is done automatically by APM so merging it in workflow is not required. Taging and sending can be done in one workflow then. 

See 

https://manual.tracardi.com/getting_started/processes/apm/

https://manual.tracardi.com/qa/how_to_configure_amp/

You can also join our slack where we help with setting up tracardi.
